squash_iterator.go

```diff
@@ -351,8 +352,8 @@ type squashRefIter struct {
 
 // NewAllRefsIter returns an iterator that will return all references
 // that match the given filters.
-func NewAllRefsIter(filters sql.Expression) RefsIter {
-	return &squashRefIter{filters: filters}
+func NewAllRefsIter(filters sql.Expression, virtual bool) RefsIter {
```

Comment:
Can you add what `virtual` means to the doc?

Reply:
Yes